src/utils/validators.js

```diff
@@ -1,6 +1,7 @@
 import _ from "lodash"
 import moment from "moment-timezone"
 
+// eslint-disable-next-line import/no-cycle
```

Comment:
why do we have a circular dependency here?

Reply:
Not too sure why, I checked the imports and we don't. Opted to disable the check here instead.

Reply:
just need change 
```
import { isLinkInternal } from "utils"
```
to 
```
import { isLinkInternal } from "utils/misc"
```
in `cspUtils.js`

Reply:
can remove this alr?